types/collection.hh

```diff
@@ -11,7 +11,7 @@
 #include <seastar/core/shared_ptr.hh>
 #include <seastar/core/sstring.hh>
 #include <vector>
-#include "types.hh"
+#include "types/types.hh"
```

Comment:
I don't think changing the includes in types/ is needed, or even good, but I guess it's a matter of style (to encourage using types/types.hh everywhere).

Reply:
yeah. i changed this file using the one-liner. i don't feel strong either way. we use two styles in this project when the included file is located in the same directory -- to include with the directory component  or just include without the directory component.

Reply:
Ok, so I'll just accept your change. I think that either approach has its pros and cons.